invenio_communities/communities/records/jsonschemas/communities/communities-v1.0.0.json

```diff
@@ -31,98 +32,123 @@
       "description": "Community identifier.",
       "type": "string"
     },
-    "title": {
-      "description": "Community title.",
-      "type": "string"
-    },
-    "description": {
-      "description": "Community description.",
-      "type": "string"
-    },
-    "type": {
-      "description": "Community type.",
-      "enum": ["organization", "event", "topic", "project"]
-    },
-    "alternate_identifiers": {
-      "type": "array",
-      "description": "Alternate identifiers for the community",
-      "uniqueItems": true,
-      "items": { "$ref": "#/definitions/identifier" }
-    },
-    "curation_policy": {
-      "description": "Community curation policy.",
-      "type": "string"
-    },
-    "page": {
-      "description": "Community page.",
-      "type": "string"
-    },
-    "website": {
-      "description": "Community website.",
-      "type": "string"
-    },
-    "domains": {
-      "description": "Community domain.",
-      "type": "array",
-      "uniqueItems": true,
-      "items": {
-        "type": "string"
-      }
-    },
-    "funding": {
-      "description": "Community funding.",
-      "type": "array",
-      "uniqueItems": true,
-      "items": {
-        "type": "string"
-      }
-    },
-    "verified": {
-      "description": "Community verified.",
-      "type": "boolean"
-    },
-    "visibility": {
-      "description": "Community visibility.",
-      "enum": ["public", "private", "hidden"]
-    },
-    "member_policy": {
-      "description": "Community member policy.",
-      "enum": ["open", "closed"]
-    },
-    "record_policy": {
-      "description": "Community record policy.",
-      "enum": ["open", "closed", "restricted"]
-    },
-    "archived": {
-      "description": "Community archived.",
-      "type": "boolean"
-    },
-    "created_by": {
-      "description": "Community creator.",
-      "type": "integer"
-    },
-    "_collections": {
+    "pid": {"$ref": "local://communities/definitions-v1.0.0.json#/internal-pid"},
```

Comment:
I guess this is duplicated here, because it's originally from `invenio-rdm-records`? Otherwise if it was in `invenio-records-resources`, we could have also e.g. just done a `/records/definitions-v1.0.0.json#/internal-pid`

Reply:
Yes exactly :)